test/oauth-server/Makefile

```diff
@@ -0,0 +1,30 @@
+SHELL = /bin/bash
+SERVICE_NAME = mocked-oauth-server
+LATEST_TAG ?= vnext
+VERSION_TAG ?= $(LATEST_TAG)-$(shell git rev-parse --short=7 --verify HEAD)
+
+default: build
+
+define build-docker-image
+	docker build \
+		--network=host \
+		--tag plgd/$(SERVICE_NAME):$(VERSION_TAG) \
+		--tag plgd/$(SERVICE_NAME):$(LATEST_TAG) \
+		--target $(1) \
+		.
+endef
+
+build-servicecontainer:
+	$(call build-docker-image,service)
+
+build: build-servicecontainer
+
+
+push: build-servicecontainer
+	docker push plgd/$(SERVICE_NAME):$(VERSION_TAG)
+	docker push plgd/$(SERVICE_NAME):$(LATEST_TAG)
+
+proto/generate:
```

Comment:
No protos to generate, lines 27 and 28 can be removed and the target can be removed from line 30